SRS.md

```diff
@@ -20,16 +20,13 @@ The best way to decrease expenses and increase profit in the restaurant business
 The proposed system will increase order accuracy and decrease the time between the customer sitting down and receiving their food. It will also decrease the time the customer has to wait to pay for their order. This should increase the overall satisfaction of the customer, promoting return business.
 
 ##### 1.2 - Scope
-To start, we are going to focus only on the customers, the servers, and the kitchen. By providing a middle-system, we will be able to speed up the 
+To start, we are going to focus only on the customers, the servers, and the kitchen. By acting as an intermediate part, we will be able to speed up the 
```

Comment:
maybe something like "...acting as an intermediate part between the restaurant staff and the customer..." might sound a bit more clear?

What do you think?

Reply:
absolutely